docs/src/main/paradox/errorhandling.md

```diff
@@ -28,6 +28,11 @@ Scala
 Java
 : @@ snip [stopping](../../test/java/sample/javadsl/StreamWrapperActor.java) { #errorHandlingSupervisor }
 
+In order to handle **Consumer fails due to WakeupExceptions** while using the ClosedGraph.
+
+Java
+: @@ snip [stopping](../../test/java/sample/javadsl/RunnableGraphStreamWrapperActor.java) { ##errorHandlingConsumerFailsWakeupExceptions }
```

Comment:
This isn't rendered correctly, perhaps because there's only a single tab here.

You can test the generated documentation with 'sbt paradox', it'll be generated in `doc/target/paradox/site/main`.